<ui version="4.0" >
 <class>ConnectDialog</class>
 <widget class="QDialog" name="ConnectDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>192</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>0</hsizetype>
    <vsizetype>0</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>386</width>
    <height>192</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>386</width>
    <height>192</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Port Connection</string>
  </property>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>7</x>
     <y>50</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Board:</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="uiLabelChan" >
   <property name="geometry" >
    <rect>
     <x>8</x>
     <y>92</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Channel Port:</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="channelCombo" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>90</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="boardCombo" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>50</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="uiLabelPrompt" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>20</y>
     <width>211</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Select the connection port:</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>207</x>
     <y>140</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="uiBtnChangeAVEnc" >
   <property name="geometry" >
    <rect>
     <x>22</x>
     <y>142</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string>Change A/V Encoders...</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>boardCombo</tabstop>
  <tabstop>channelCombo</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConnectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConnectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
